test/checks/language/xml-lang-mismatch.js

```diff
@@ -0,0 +1,71 @@
+describe('xml-lang-mismatch', function() {
+	'use strict';
+
+	var node;
+	var fixture = document.getElementById('fixture');
+	var checkContext = axe.testUtils.MockCheckContext();
+
+	beforeEach(function() {
+		node = document.createElement('html');
```

Comment:
You could use a div instead, so you don't get into browser support trouble. The check is agnostic to what node its testing.